JointPosition[] joint_trajectory
GripperPosition[] gripper_trajectory

